.eslintrc.yaml

```diff
@@ -0,0 +1,71 @@
+# This is a baseline set of rules intended for usage in any ES6-based project.
+#
+# Be sure to add your project's specific environment(s) as needed via external configuration.
+#   See: http://eslint.org/docs/user-guide/configuring#specifying-environments
+
+---
+rules: # http://eslint.org/docs/rules
+
+  # Possible Errors
+
+  valid-jsdoc: [1]
+  curly: [2]
+  dot-notation: [2]
+  eqeqeq: [2]
+  no-div-regex: [2]
+  no-eval: [2]
```

Comment:
So, I get that it's more...strict...and enforces a lot (maybe all) of the rules that we're already enforcing with ESLint, but where are you seeing that it's more performant? I can't find anything definitive there.

FWIW this actually runs *slower* for me w. `use strict`: 
https://jsperf.com/es5-use-strict-performance/2